docker-compose.yml

```diff
@@ -12,7 +12,8 @@ services:
       '--log_bin=binlog',
       '--max_binlog_size=8M',
       '--binlog_format=row',
-      '--server-id=1'
+      '--server-id=1',
+      '--binlog_rows_query_log_events=ON'
```

Comment:
You must add this to .github/workflows/tests.yml:27 to make tests work or skip test if mysql variable is not set.

Reply:
Oh! Thanks! The github pipeline just runs when the review is given. But thanks ... i'll add it :)